adventure-wise/src/my-trips/Trips.jsx

```diff
@@ -0,0 +1,17 @@
+import TripDetails from "./components/TripDetails";
+import TripList from "./components/TripList";
+import "./Trips.css";
+
+const Trips = () => {
+  return (
+    <div className="container">
```

Comment:
Change classnames to be more unique or different to prevent any integration issues. ie. trip-container, etc